compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_loops_for_vectorization.mlir

```diff
@@ -140,30 +140,6 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
     return %0 : !t3_f32
   }
 
-  // Check that the final 3 dimensions do have the pattern of a matmul (or matmul transpose)
```

Comment:
Why does this test need to be removed?

Reply:
You are right, this shouldn't be removed. I'll just modify the checks.